### PR TITLE
View these Examples

### PR DESCRIPTION
**Pull request by DragonMoffon**
SO MANY EXAMPLES

I gave up at the GL examples. I also didn't touch the platformer tutorial or any of the doc tutorials.

### CODE REVIEW

File: arcade/examples/gui/0_basic_setup.py
Comment (eruvanos) [inline arcade/examples/gui/0_basic_setup.py:1]: should the main be called?

File: arcade/gui/view.py
Comment (eruvanos) [inline arcade/gui/view.py:1]: Why did you remove the background?
Isn't it also used in all the UI examples?

Comment (DragonMoffon) [reply arcade/gui/view.py:1]: Ah because I added the background as a property of view

File: arcade/examples/gui/0_basic_setup.py
Comment (DragonMoffon) [reply arcade/examples/gui/0_basic_setup.py:1]: yeah :p

File: arcade/gui/view.py
Comment (eruvanos) [reply arcade/gui/view.py:1]: Maybe I oversaw that ... mh .... found it, you are changing the window background_color via the property 🤔
Not sure, in that case the `on_show_view` method would have to set background everytime again.

My idea was to have a per view property, that does not require `on_show_view code`

Comment (DragonMoffon) [reply arcade/gui/view.py:1]: Yeah you're right I will update View to have this behaviour
